modules/tensor_mechanics/src/materials/ComputeLinearElasticStress_CUT_DT.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeLinearElasticStress_CUT_DT.h"
+
+template<>
+InputParameters validParams<ComputeLinearElasticStress_CUT_DT>()
+{
+  InputParameters params = validParams<ComputeStressBase>();
+  params.addClassDescription("Compute stress using elasticity for small strains");
+  return params;
```

Comment:
This class description looks like it was copied from the `ComputeLinearElasticStress` class; would you please add into the description the difference between the two classes, including a brief reason why a user would want to use this linear elastic stress class instead of the original?